### PR TITLE
PaperUI: Install extensions by dropping url

### PR DESCRIPTION
**Pull request by aounhaider1**
Allows installing extensions by dropping url from eclipse marketplace. Should be merged after support is added to server side for installing extensions from url.

Signed-off-by: Aoun Bukhari <bukhari@itemis.de>

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.extension.js [hunk 1]
```diff
@@ -110,6 +120,15 @@ angular.module('PaperUI.controllers.extension', [ 'PaperUI.constants' ]).control
         $scope.masonry(true);
     });
 
+    $scope.linkDropped = function(ev) {
```
Comment (htreu) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.extension.js]: does `$scope.linkDropped` even get called anywhere?

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.extension.js [hunk 2]
```diff
@@ -110,6 +120,15 @@ angular.module('PaperUI.controllers.extension', [ 'PaperUI.constants' ]).control
...
+        ev.preventDefault();
+        var data = ev.dataTransfer.getData("text");
+        alert(data);
+    }
+    $scope.acceptDrop = function(ev) {
```
Comment (htreu) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.extension.js]: does `$scope.acceptDrop` get called?

**Comment by aounhaider1** [comment]
Updated. Note that server side support for installing extensions is still not there. Should be merged ONLY after its supported.

**Comment by htreu** [comment]
Server side support for extensions installed via URLs from eclipse marketplace: #3232.

**Comment by htreu** [comment]
Superseded by #3258 which will also include the server side code and therefore fix #3232.

**Comment by htreu** [comment]
@kaikreuzer pls close this one.

**Pull request closed without merging**